manticore/ethereum.py

```diff
@@ -1272,6 +1272,13 @@ def flagged(flag):
                 logger.debug("Using iterpickle to dump state")
                 statef.write(iterpickle.dumps(state, 2))
 
+        with testcase.open_stream('trace') as f:
```

Comment:
Search for "runtime_coverage" I think you'll need similar trick to separate init from runtime trace?